official/resnet/resnet.py

```diff
@@ -264,10 +363,11 @@ def projection_shortcut(inputs):
 
   # Only the first block per block_layer uses projection_shortcut and strides
   inputs = block_fn(inputs, filters, training, projection_shortcut, strides,
-                    data_format)
+                    data_format, version=version)
```

Comment:
Again, not sure if it's feasible, but not having to pass the version all the way down would keep things cleaner, it seems.

Reply:
Indeed, this was a side effect of moving block_fn selection into the init.